metricbeat/internal/metrics/cpu/metrics_procfs_common.go

```diff
@@ -39,8 +41,75 @@ func Get(procfs resolve.Resolver) (CPUMetrics, error) {
 		return CPUMetrics{}, errors.Wrapf(err, "error opening file %s", path)
 	}
 
-	return scanStatFile(bufio.NewScanner(fd))
+	metrics, err := scanStatFile(bufio.NewScanner(fd))
+	if err != nil {
+		return CPUMetrics{}, fmt.Errorf("scanning stat file: %w", err)
+	}
+
+	infoPath := procfs.ResolveHostFS("/proc/cpuinfo")
+	infoFd, err := os.Open(infoPath)
+	if err != nil {
+		return CPUMetrics{}, fmt.Errorf("opening /proc/cpuinfo: %w", err)
```

Comment:
small nit, the error message should use the `infoPath` variable, since if someone is using a custom HostFS, just printing `/proc/cpuinfo` will be confusing.

Reply:
Thanks for catching that! Fixed on: 01d58fed86